clang/lib/Basic/SourceManager.cpp

```diff
@@ -2227,6 +2227,28 @@ LLVM_DUMP_METHOD void SourceManager::dump() const {
   }
 }
 
+static std::string NumberToHumanString(uint64_t Number) {
```

Comment:
LLVM Style guide asks to use `camelCase` and a verb phrase (see https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly). `Number` also seems a bit redundant, because it's the first parameter anyway.

Let's turn this to `camelCase`?
Extra suggestion (optional): Maybe rename to `toHumanString`?

Reply:
Renamed to `humanizeNumber()`.